fdbrpc/FlowTransport.actor.cpp

```diff
@@ -428,7 +428,19 @@ ACTOR Future<Void> connectionKeeper( Reference<Peer> self,
 				self->lastConnectTime = now();
 
 				TraceEvent("ConnectingTo", conn ? conn->getDebugID() : UID()).suppressFor(1.0).detail("PeerAddr", self->destination);
-				Reference<IConnection> _conn = wait( timeout( INetworkConnections::net()->connect(self->destination), FLOW_KNOBS->CONNECTION_MONITOR_TIMEOUT, Reference<IConnection>() ) );
```

Comment:
why not just wrap the original `wait()` into a try-catch?

Reply:
I was trying to handle scoping problems with _conn, but ended up with the same problem anyways. I think the current attempt now works.